### DISCUSSION TITLE
Saturday as Start of week

### DISCUSSION THREAD

**Discussion by 2u841r** [ActivityWatch/activitywatch#1058] [Ideas]
![activityW](https://github.com/ActivityWatch/activitywatch/assets/77893877/94e8fc2b-f048-49f3-957e-047fb93e1102)

According to Time and Date, Iran, Afghanistan and Somalia (also Bangladesh) start their week on Saturday. In some Middle Eastern countries, such as Saudi Arabia and the United Arab Emirates, Saturday is the first day of the week according to the Islamic calendar. This is because Friday is considered the holiest day of the week in Islam, and is the day of congregational prayer (Jumu'ah).

**Comment by ErikBjare** [comment]
Ah right! I'll add that right away.

**Reply by ErikBjare** [reply]
Added in https://github.com/ActivityWatch/aw-webui/commit/80db0063e6c63b8bccceeaefa3885e74c741049c
